ui/raidboss/data/05-shb/raid/e9s.js

```diff
@@ -538,7 +538,7 @@ export default {
       durationSeconds: (data) => data.finalArtOfDarkness ? 16 : 9,
       alertText: (data, _matches, output) => {
         // Perform the callout now, regardless if it's The Second or Third Art Of Darkness
-        callout = data.artOfDarkness.slice();
+        const callout = data.artOfDarkness.slice();
```

Comment:
Can you explain how this fixes #2995?  It seems like even if `callout` is a global here, that it gets clobbered in this function and so I'm not sure I follow how the behavior changes by adding `const`.

Reply:
Except that it's not a global, it's a local, and it's being used without being declared.

![image](https://user-images.githubusercontent.com/6119598/120682097-0edeec00-c46a-11eb-97ca-c40449ecc5ae.png)

Reply:
Oh, it's `use strict` that makes this an error.  Got it.